app.js

```diff
@@ -34,3 +34,18 @@ openToast.addEventListener('click',()=>{
         toastMsg.style.display="none";
     },3000)
 })
+
+
+//hamburger-menu
+const menuBtn = document.querySelector(".hamburger-menu-btn");
+const menuLinks = document.querySelector(".hamburger-menu-links");
+
+menuBtn.addEventListener('click',()=>{
+    if (menuLinks.style.display == "block"){
+        menuBtn.innerHTML = '<i class="fa fa-bars"></i>';
+        menuLinks.style.display = "none";
+    }else{
+        menuBtn.innerHTML = '<i class="fa fa-close"></i>';
+        menuLinks.style.display = "block";

```

Comment:
I think this piece of code is not working

Reply:
I did check and it seems to be working, can you mention what exactly is not working so I can take a look.

Reply:
Yea, it works! When I was checking the hamburger menu, that close icon was not appearing after you open it. Maybe it didn't load properly idk, but anyway it's solved!

Reply:
Oh okay, thanks :D